multiversx_sdk_cli/cli_shared.py

```diff
@@ -103,13 +103,13 @@ def add_tx_args(
             type=int,
             required=False,
             default=None,
-            help="# the nonce for the transaction",
+            help="# the nonce for the transaction. The nonce is fetched from the network if --proxy is provided.",
```

Comment:
_the nonce for the transaction. If not provided, is fetched from the network_ (we can omit the information about the presence of `--proxy`, since the user will be warned on bad usage, anyway.